ete3/ncbi_taxonomy/ncbiquery.py

```diff
@@ -142,15 +142,17 @@ def get_fuzzy_name_translation(self, name, sim=0.9):
         :return: taxid, species-name-match, match-score
         '''
 
-
-        import sqlite3.dbapi2 as dbapi2
+        try:
+	    import pysqlite2.dbapi2 as dbapi2
```

Comment:
Which version of python are you using to require `pysqlite2`?
`sqlite3` is part of the standard library since Python 2.5 which has long lost support.

Reply:
As described [here](https://stackoverflow.com/questions/1545479/force-python-to-forego-native-sqlite3-and-use-the-installed-latest-sqlite3-ver#1546162) the version numbers of pysqlite are complicated.

Reply:
The reason for importing sqlite2 was, if I recall correctly, to keep support for the SQLite-Levenshtein module (which was only available for sqlite2 some years ago)...

Reply:
exactly. this is still the case. the function enable_load_extension is not available in pysqlite by default.